### PR TITLE
Add a Help section to the Sphinx docs

### PR DESCRIPTION
**Pull request by ahill187**
Resolves: https://github.com/ivadomed/ivadomed/issues/568

1. Add a `Help` section under `Getting Started` in the docs.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Ivadomed Docs
```
Comment (jcohenadad) [inline docs/README.md:1]: out-of-scope? or maybe it includes a recently-merged branch?

Comment (ahill187) [reply docs/README.md:1]: Ah yes, I added the README...should I make a separate PR for that?

Comment (jcohenadad) [reply docs/README.md:1]: isn't the installation covered in https://github.com/ivadomed/ivadomed/pull/561/files?

Comment (ahill187) [reply docs/README.md]: The installation doesn't go over how to build the `Sphinx` docs, and the `requirements.txt` is missing one of the `Sphinx` dependencies.

Comment (ahill187) [reply docs/README.md]: I have removed the `README` from this pull request though

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**